### ISSUE TITLE
Specific versions require @DependsOn("amqpAdmin") on the @Component which defines the @RabbitListener

### ISSUE THREAD

**Issue by abedurftig** [freenowtech/spring-multirabbit#69]
We have found that starting with a specific Spring Boot and Spring Rabbit version you require a particular annotation, for the declarations in the @RabbitListener annotation to be executed.

| SpringBoot        | SpringCoud           | spring-rabbit  | Status
| ------------- |-------------| -----|---
| 2.4.0      | 2020.0.0 | 2.3.1  | OK 
| 2.4.2      | 2020.0.3      |   2.3.4 | OK 
| 2.4.3 | 2020.0.3      |    2.3.5 | NOK (requires @DependesOn)

Example:
```
@Component
@DependsOn(value = ["amqpAdmin"])
class RabbitMqEventListener {
  @RabbitListener
  fun consume() {}
}
```

**Comment by rwanderc** [comment]
This should have been fixed with https://github.com/freenowtech/spring-multirabbit/commit/ffe989047dd5426095460ea0626d797f6d5dd594. Would you pls advise if this is still happening?

**Comment by athlordJojo** [comment]
Hi wander! I have the same problem , but with version 2.7.0 of multi rabbit and spring boot 2.7.3
Is this a known issue ?

**Comment by athlordJojo** [comment]
Hi Wander! 

 I get the this error also with version 2.7.0 of multi rabbit in spring boot 2.7.3.

Is this a known issue ?

**Comment by rwanderc** [comment]
Hi @athlordJojo, 👋
Well.... it should have been fixed back in 2.4.x. 🤔
Could you share some code to showcase the error?

**Comment by athlordJojo** [comment]
I will try to get some time and try to create a reproducer.
